### ISSUE TITLE
Can we have the current archon quest.

### ISSUE THREAD

**Issue by Elvaranna** [seriaati/genshin.py#184]
I see that on the apps, when looking at the note i can find the which par of the archon quest i am at. since i am running several accounts in parallel, it would be really nice to have that in the get_genshin_note() method.
Do you think it is possible to add?

**Comment by jokelbaf** [comment]
Implemented on the `dev` branch in #185.

```
pip install git+https://github.com/thesadru/genshin.py.git@dev
```
